File: Responder/Responder/Commands/QueryStatisticsCommand.cs
namespace Responder.AppConfig
{
    public class QueryStatisticsCommand : ICommand
    {
        #region Properties
        public CommandType Type { get; private set; }
        public int Length { get; set; }
        #endregion

        #region Constructors
        public QueryStatisticsCommand()
        {
            Type = CommandType.QueryStatistics;
        }
        #endregion
    }
}

File: Imperium/src/com/imperium/Commands/QueryStatisticsCommand.java
package com.imperium.Commands;

public class QueryStatisticsCommand implements ICommand{
    private CommandType type;
    private int length;

    public CommandType getType() {
        return type;
    }

    public int getLength() {
        return length;
    }

    public void setLength(int length) {
        this.length = length;
    }

    public QueryStatisticsCommand(){
        type = CommandType.QueryStatistics;
    }
}
